autodiff/reverse/var/var.hpp

```diff
@@ -154,92 +211,149 @@ ExprPtr<T> constant(const T& val);
 //------------------------------------------------------------------------------
 // ARITHMETIC OPERATORS (DECLARATION ONLY)
 //------------------------------------------------------------------------------
-template<typename T> ExprPtr<T> operator+(const ExprPtr<T>& r);
```

Comment:
Here too - keep on same line (otherwise it gets very noisy).

Reply:
These changes are also making it difficult for me to recognize any possible new addition among the formatting changes.